Source/WarpX.cpp

```diff
@@ -237,8 +237,7 @@ WarpX::WarpX ()
 
 WarpX::~WarpX ()
 {
-    int nlevs_max = maxLevel() +1;
```

Comment:
I think that temporary variable is intentional, since it avoids to call `maxLevel()` for each run through the loop.
But we should make `nlevs_max` constant :)
```C++
int const nlevs_max = maxLevel() + 1;
```

Reply:
Good point!  Would `const int nlevs_max = maxLevel() + 1;` work as well?

Reply:
Of course, I just fell in love with [east const](https://mariusbancila.ro/blog/2018/11/23/join-the-east-const-revolution/) since it's so nicely consistent.